WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -1115,10 +1115,18 @@ def finalize(self):
     """Perform any final actions to clean up after job execution.
     """
     self.log.info('Running JobWrapper finalization')
+
     # find if there are pending failover requests
     requests = self.__getRequestFiles()
     outputDataRequest = self.failoverTransfer.getRequest()
-    requestFlag = len(requests) > 0 or not outputDataRequest.isEmpty()
+
+    # try to send the failover request
+    res = self.sendFailoverRequest()
+    if not res['OK']:  # This means that the request could not be set
+      self.failedFlag = True
+    failoverRequests = res.get('Value')
+
+    requestFlag = failoverRequests or len(requests) > 0 or not outputDataRequest.isEmpty()
```

Comment:
I am afraid that this screws up the following job status reports. If sending the request fails, this will result in Failed/Pending Requests, which is obvious ly wrong. I think we have to have an additional flag here then: failedRequestFlag (for the failure of sending requests) and then examine all the combinations to make the status report. For example, failedRequestFlag and not failedFlag and requestFlag results in Failed job and logging message: Job finished successfully but failed sending failover requests.

Reply:
It's probably correct what you suggest. I will have a deeper look later; this basically adds a new `MinorStatus: "Failed setting requests"` (? maybe it exists already... I will check).

Reply:
I changed it. Please have a look.